src/main/java/hudson/remoting/jnlp/Main.java

```diff
@@ -104,6 +104,25 @@ public class Main {
             usage="If the connection ends, don't retry and just exit.")
     public boolean noReconnect = false;
 
+    @Option(name = "-delay",
```

Comment:
what do you think about calling this `retryDelay`? The existing name seems to be to imply a delay for startup
```suggestion
    @Option(name = "-retryDelay",
```

Reply:
I was considering that name in the first place anyway, so this sounds good.